src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Form/Entity/AttributeTypeFactory.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
```

Comment:
Enable strict types?

Reply:
Yes, I will add this to the next stage (commands)